nncf/common/accuracy_aware_training/training_loop.py

```diff
@@ -201,11 +203,12 @@ class AdaptiveCompressionTrainingLoop(BaseEarlyExitCompressionTrainingLoop):
     def __init__(self,
                  nncf_config: NNCFConfig,
                  compression_controller: CompressionAlgorithmController,
+                 original_model_accuracy: float,
                  lr_updates_needed=True, verbose=True,
                  minimal_compression_rate=0.0,
                  maximal_compression_rate=0.95,
                  dump_checkpoints=True):
-        super().__init__()
+        super().__init__(compression_controller)
```

Comment:
I would like to know your motivation for these changes? if you are leaving `self.compression_controller = compression_controller` on the next line.

Reply:
Forgot to remove the next line.